bin/eslint.js

```diff
@@ -232,4 +231,12 @@ function analyzeFiles() {
   }
 }
 
-analyzeFiles();
+if (validateConfig(options.configFile)) {
+  console.error("ESLint is running with the " + cli.getConfigForFile(null).parser + " parser.");
+
+  analyzeFiles();
+} else {
+  console.error("No rules are configured. Make sure you have added a config file with rules enabled.");
+  console.error("See our documentation at docs.codeclimate.com/docs/eslint for more information.");
```

Comment:
@wfleming yep sounds good

@ABaldwinHunter I think there are too many possibilities to list here (`.eslintrc(.js|.json|.yml.yaml|nothing`) and it would be a bit wordy. My link to the docs was so that people would see it there